docker-app/qfieldcloud/core/views/users_views.py

```diff
@@ -146,3 +168,235 @@ class ListUserOrganizationsView(generics.ListAPIView):
 
     def get_queryset(self):
         return Organization.objects.of_user(self.request.user)
+
+
+class TeamMemberDeleteView(APIView):
+
+    def delete(self, request, organization_name, team_name, member_username):
+        """
+        DELETE /organizations/<str:organization_name>/team/<str:team_name>/members/<str:member_username>/
+        Remove a specific member from a team within an organization.
+        """
+        organization = get_object_or_404(Organization, username=organization_name)
+        team_name = f'@{organization_name}/{team_name}'
+        team = get_object_or_404(Team, team_organization=organization, username=team_name)
+        member = get_object_or_404(User, username=member_username)
+
+        # Check if the member belongs to the team
+        team_member = get_object_or_404(TeamMember, team=team, member=member)
+
+        # Remove the member
+        team_member.delete()
+
+        serializer = DeleteMemberSerializer(data={'message': f"Member {member.username} removed from team {team.username}"})
+        return Response(serializer.data, status=status.HTTP_204_NO_CONTENT)
```

Comment:
How come we both say: 204 no content, but we pass the `message` in the response? I would suggest to drop the `message` part, it should be just empty content if all other endpoints do that (double check if this is correct memory).

Reply:
Message Deleted!